src/OrbitGl/TrackManager.cpp

```diff
@@ -43,8 +48,9 @@ TrackManager::TrackManager(TimeGraph* time_graph, orbit_gl::Viewport* viewport,
       layout_(layout),
       capture_data_{capture_data},
       app_{app} {
-  GetOrCreateSchedulerTrack();
-  tracepoints_system_wide_track_ = GetOrCreateThreadTrack(orbit_base::kAllThreadsOfAllProcessesTid);
```

Comment:
I don't like that we are erasing this Create method and we aren't inserting it in any place. It is one step behind from the splitting between Get and Create.

Reply:
I think this is debatable. IMO if we split create / get, this problem will go away anyways because we have a clear separation. Right now, it's in a bit of an odd state - my initial unit tests failed because I was not realizing that the `TrackManager` is implicitly already creating tracks, and I decided to make the creation a responsibility of `TimeGraph`.

I think this depends a bit on what your plans were regarding track creation and where it should take place. I'm fine with bringing this back into `TrackManager`. Then I think it should be consistent though (meaning the scheduler track should be created when the first timer arrives, as are all the other tracks). Maybe we should briefly discuss this offline.

Reply:
My point was why we are not inserting it in TimeGraph similar to the scheduler one.

Reply:
Ah, yes, that a very valid point. It's similar to the other tracks though, because they are created when data arrives. But since we agree that the responsibility of creating tracks will move out of `TrackManager` soon, so I would leave it like it is in this PR, WDYT?